src/components/NewsFeedHeader/NewsFeedHeader.tsx

```diff
@@ -0,0 +1,10 @@
+// import styles from './NewsFeedHeader.module.scss';
+
+const NewsFeedHeader = () => {
+  return (
+    <div>
+    </div>
+  );
+};
+
+export default NewsFeedHeader;
```

Comment:
plz add empty line end of file

Reply:
> please add unit test

Can't add unit test without merging your PR yet